src/frontend/pages/SignInPage/SignInPage.css

```diff
@@ -0,0 +1,35 @@
+.auth-screen {
+  margin-left: 1rem;
+  margin-right: 1rem;
```

Comment:
You can use margin : 0 1rem; here

Reply:
I have also used `margin-top`. I will use the short-hand property here. Thank you!